build-system/tasks/storybook-react/react-env/webpack.config.js

```diff
@@ -0,0 +1,75 @@
+const path = require('path');
+const {BUILD_CONSTANTS} = require('../../../compile/build-constants');
+const {DefinePlugin} = require('webpack');
+const {getRelativeAliasMap} = require('../../../babel-config/import-resolver');
+const {webpackConfigNoChunkTilde} = require('../../storybook/env-utils');
+
+const rootDir = path.join(__dirname, '../../../..');
+
+module.exports = ({config}) => {
+  config.resolveLoader = {
+    modules: [
+      path.join(__dirname, '../node_modules'),
+      path.join(rootDir, 'node_modules'),
+    ],
+  };
+  config.resolve = {
+    modules: [
+      path.join(__dirname, '../node_modules'),
+      path.join(rootDir, 'node_modules'),
+    ],
+    alias: {
+      // Alias the accordion
+      '../../../amp-accordion/1.0/component':
+        '../../../amp-accordion/1.0/dist/component-react.max',
+      // Alias the base carousel
+      '../../../amp-base-carousel/1.0/component':
+        '../../../amp-base-carousel/1.0/dist/component-react.max',
+      // Alias the base carousel styles
+      '../../../amp-base-carousel/1.0/component.jss':
```

Comment:
@alanorozco WDYT about adding `#extensions` as a root alias at this point?

Reply:
These paths are ugly for sure. But I'm not sure that making them prettier with an alias is the answer. 

The specific items in dist list can go away because there's a pattern. A [`ResolverPlugin`](https://github.com/webpack/docs/wiki/list-of-plugins#resolverplugin) can be configured to follow it:

```
${x}/component -> ${x}/dist/component-react.max
${x}/component.jss -> ${x}/dist/styles.css
```